<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Info</class>
 <widget class="QWidget" name="Student_Info">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="top_frame_7">
     <property name="minimumSize">
      <size>
       <width>471</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #E07A5F;
color: #3D405B; </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_mcq_heading">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>58</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Kalpurush</family>
          <pointsize>26</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
font: 26pt &quot;Kalpurush&quot;;</string>
        </property>
        <property name="text">
         <string>পরিচয়</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>-10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>451</width>
       <height>311</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(243, 240, 221);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>7</number>
      </property>
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Kalpurush</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(224, 122, 95);
color: #000;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>নাম</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lbl_std_name">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 3px solid #E07A5F;
background-color:#3D405B;
border-radius: 10px;
color: rgb(255, 170, 127);
padding-left: 15px;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lbl_std_image">
        <property name="styleSheet">
         <string notr="true">border: 3px dashed rgb(0, 43, 91);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Images/gallery2.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>131</width>
          <height>51</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Kalpurush</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(224, 122, 95);
color: #000;
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>আইডি</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lbl_std_id">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>51</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 3px solid #E07A5F;
background-color:#3D405B;
border-radius: 10px;
color: rgb(255, 170, 127);
padding-left: 15px;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_submit">
        <property name="minimumSize">
         <size>
          <width>309</width>
          <height>48</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Kalpurush</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
border: 3px solid #E07A5F;
background-color:#3D405B;
border-radius: 20px;
color: #fff;
}

QPushButton:hover:!pressed {
background-color: #E07A5F;
color: #fff;
border: 3px solid #3D405B;
}

QPushButton:checked {
        background-color:#3D405B; /* Selected state color */
        color: white;
}</string>
        </property>
        <property name="text">
         <string>সাবমিট করুন</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
